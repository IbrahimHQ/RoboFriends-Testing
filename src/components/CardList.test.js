import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import CardList from './CardList';

it('renders CardList', () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Smith",
      username: "John",
      email: "john@gmail.com"
    }
  ]

  const wrapper = shallow(<CardList robots={mockRobots} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});