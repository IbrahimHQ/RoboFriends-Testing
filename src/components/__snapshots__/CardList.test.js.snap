// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CardList 1`] = `
<div>
  <Card
    email="john@gmail.com"
    id={1}
    key="0"
    name="John Smith"
  />
</div>
`;
